packages/manager/modules/pci/src/projects/project/storages/objects/objects.routing.js

```diff
@@ -25,17 +25,20 @@ export default /* @ngInject */ ($stateProvider) => {
       viewContainer: /* @ngInject */ ($state, projectId) => (container) =>
         $state.go('pci.projects.project.storages.objects.object', {
           projectId,
-          containerId: container.id,
+          containerId: container.id || container.name,
```

Comment:
I'm not sure it is a good idea to start either having `id` or `name`. Especially it will require to make some adaptation to find id depending on name in the used component and it starts to get tricky

Reply:
Well this is a bit strange to see i agree but with swift they had id and name and faced some difficulties.
There is no ID(openIO) either. Containers have no ID, they only have a unique name. For swift, we used to compute an ID using name and region (base64) but this was dirty and led to issue. We do not want to reproduce this with OpenIO.

Reply:
Ok so maybe we should abstract with those kind of containers that the id is the name (with a model for instance)